contracts/exchange/test/signature_validator.ts

```diff
@@ -236,6 +236,18 @@ blockchainTests.resets('MixinSignatureValidator', env => {
             return expect(tx).to.revertWith(expectedError);
         });
 
+        it('should revert when signer is an EOA and SignatureType=Wallet', async () => {
```

Comment:
Thanks for adding this test case. Another that we should probably add is: `should revert when signer is a contract that returns the empty bytes (or true) and SignatureType=Wallet`